### ISSUE TITLE
sessions: create and delete should use nested transaction

### ISSUE THREAD

**Issue by jirikuncar** [inveniosoftware/invenio-accounts#98]
- [ ] do not use Flask-Security datastore for managing SessionActivity
- [ ] verify that commit is called at after request when session is created/deleted

**Comment by jirikuncar** [comment]
Example:
```python
from invenio_db import db
from invenio_accounts.models import SessionActivity

with db.session.begin_nested():
     sa = SessionActivity.query.filter(...).one()
     sa.foo = "bar"
     db.session.add(sa)
```

**Comment by jirikuncar** [comment]
Closed via 8ba9b17a74ef257ca67c5e1c7f9e1c3de991e8d0
